docs/development/doc_guide.rst

```diff
@@ -61,3 +59,17 @@ Narrative Documentation
 
 * Each subpackage must have narrative documentation describing its
   use.
+
+Substitutions
+=============
+
+The `common_links.txt <https://github.com/PlasmaPy/PlasmaPy/blob/master/docs/common_links.txt>`_
```

Comment:
```suggestion
The `common_links.rst
<https://github.com/PlasmaPy/PlasmaPy/blob/master/docs/common_links.rst>`_
```

You can actually split the hyperlink like this on two or more lines and it will still work.  This was really nice when I discovered it.